### ISSUE TITLE
Cannot connect to any database!!

### ISSUE THREAD

**Issue by safiullahvirk** [mzmine/mzmine2#395]
Hello Everyone, 
I am using mzmine2, 29, it cannot connect to online databases for peak identification. Can someone help me to resolve this issue? Thank you,

**Comment by tsufz** [comment]
Hello,  it would helpful if you could specify the databases you cannot reach, please. They may be down for service or changed interface conditions which has to fixed.

Thanks
Tobias

**Comment by safiullahvirk** [comment]
none of the online databases in the default list of mzmine,

**Comment by tomas-pluskal** [comment]
See How to report issues: https://mzmine.github.io/support.html

**Comment by safiullahvirk** [comment]
I am using MzMine 2.29 (Windows 7 64-Bit),
Java version 8 (build 1.8.0_151-b12) 
R for windows 3.4.2

![issue 2](https://user-images.githubusercontent.com/34440385/34068667-41db0b32-e27a-11e7-9044-deba60ef553c.jpg)
![issue 3](https://user-images.githubusercontent.com/34440385/34068668-420f20ca-e27a-11e7-913d-1a1f78bb9045.jpg)
![issue 1](https://user-images.githubusercontent.com/34440385/34068666-41a7f224-e27a-11e7-9d47-98cef8eb0f45.jpg)
![Uploading issue 2.jpg…]()
![Uploading issue 3.jpg…]()

**Comment by tomas-pluskal** [comment]
Do you use proxy for internet connection? Have you tried connecting from other places? E.g., using some public WiFi.

**Comment by safiullahvirk** [comment]
Yeah, i tried different networks without proxy, just use normal internet, public and private,

**Comment by pieytremblay** [comment]
i have the same problem here, using HMDB database.

I tried 3 different computers and mzmine v.2.26, 2.29, 2.31 and 2.32

java 8 1.8 update 161
![bug mzmine](https://user-images.githubusercontent.com/38320262/38677476-5fb3bd4e-3e2c-11e8-844e-783cb07e15e5.png)

**Comment by tomas-pluskal** [comment]
@pieytremblay This does not look like the same problem, because the error message that you see is different from the the errors reported above. Do you see this with all databases or only with HMDB?

**Comment by tomas-pluskal** [comment]
It seems like a problem on HMDB side - for example this URL works:
http://www.hmdb.ca/metabolites/HMDB0000538.xml
but this one throws a server error:
http://www.hmdb.ca/metabolites/HMDB0059593.xml

**Comment by tomas-pluskal** [comment]
Also this throws an error: http://www.hmdb.ca/metabolites/HMDB0059593

**Comment by tomas-pluskal** [comment]
FYI, the HMDB folks just fixed the issue.

**Comment by tomas-pluskal** [comment]
Nevertheless, I had to update the HMDB search module in MZmine, because they have changed the input format. The fix will be included in next MZmine version (https://github.com/mzmine/mzmine2/commit/d38951de99299e9cd79ecc31a84577abca54b8ca).

**Comment by shweta-talreja** [comment]
Hello there,

Since I am facing related problem, so thought of continuing my issue in this thread.

I am using MZmine 2.32 (windows 10 64-bit).
and tried both java 8 and 9.

With java 8 was unable to connect to online database(pubchem). However, with java 9 (JDK) was able to connect through but after reaching 25-28% it suddenly shows error. 
I have tried through different servers. 

I dont know what to do. Plz help!!
